Fix up custom certificate documentation (#1154)
panther-labs/panther: docs/gitbook/operations/advanced-setup/configuring-custom-domains.md

Out of the box, Panther ships with a self-signed certificate generated at deployment time. While this setup is better than not having SSL/TLS enabled at all on the web server, it is still far from best practice especially for a security tool. Panther *strongly* recommends you replace this self-signed certificate with a proper certificate before using Panther in a production environment.

> Setting up a custom domain for the Panther app is fairly easy, and takes four steps:

Reviewer: ```suggestion
To set up a custom domain for Panther, follow these easy four steps:
```

  1. Register a domain
  2. Get a signed certificate for your domain into AWS
  3. Configure Panther
  4. Setup an alias from the domain to the auto-generated load balancer URL
  

> All of these steps can be completed from within the AWS ecosystem. This guide will walk through configuring a custom domain totally within the AWS ecosystem, and should take ~45 minutes and cost less than $20/year. If you prefer to manage your certificates outside of AWS the same steps listed below should still apply, although the exact details will depend on where you are registering your certificate.

Reviewer: ```suggestion
All of these steps can be completed from within the AWS ecosystem. This guide will walk through configuring a custom domain totally within the AWS ecosystem and should take ~45 minutes and cost less than $20/year. If you prefer to manage your certificates outside of AWS, the steps listed below still apply, although the exact details will depend on where you are registering your certificate.
```

### Register a domain

In this guide we'll walk through registering a domain through AWS Route53, although any domain registrar should work. If you already have a domain registered, or perhaps an internal CA that manages certificates for your organization, this step can be skipped.

>   1. Navigate to the [Route53](https://console.aws.amazon.com/route53/home) console in AWS and click the `Registered domains` tab.

Reviewer: ```suggestion
  1. Navigate to the [Route53](https://console.aws.amazon.com/route53/home) console and click the `Registered domains` tab.
```

Reviewer: No need to say 'in AWS', it's implied (there's another occurrence below)

  
### Configure Panther

> At this point, it is time to configure Panther to use your custom certificate and domain. This can be done to either an active Panther deployment or to a new Panther deployment.

Reviewer: ```suggestion
The next step is to configure Panther to use your new certificate and domain. This can be completed with either an active Panther or a new Panther deployment.
```

> For pre-packaged deployments, navigate to the [CloudFormation](https://console.aws.amazon.com/cloudformation/home) console in AWS. From here, find the panther master stack (you named this stack when deploying panther, most likely it is just called `panther`). Select this stack, then click the `Update` button. Ensure the `Use current template` option is selected and click `Next`. On the next page, find the `Parameters` section and update the following two parameters:

Reviewer: ```suggestion
For pre-packaged deployments, navigate to the [CloudFormation](https://console.aws.amazon.com/cloudformation/home) console. From here, find the Panther master stack (called `panther` by default), select this stack, and click the `Update` button. Ensure the `Use current template` option is selected and click `Next`. On the next page, find the `Parameters` section and update the following two parameters:
```

Reviewer: These steps should be broken out into an ordered list

  - `CertificateArn` - in this field, put the full ARN of the ACM certificate created in step two. This can be retrieved from the ACM console.
  - `CustomDomain` - in this field, put the domain name you registered in step one.
  

> Click the `Next` button until you reach the final `Review` step. Double check that the fields entered above are correct, then click the `Update stack` button. You may need to check the `I acknowledge that AWS CloudFormation might...` check boxes. After clicking `Update stack`, panther will update with your new certificate. An update should only take a few minutes.

Reviewer: Do they have to check the capabilities box? or not. I'd make the language more definitive if so
